OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/model/Orginazation.java

```diff
@@ -0,0 +1,14 @@
+/*

```

Comment:
what does this class do?

Reply:
As of right now, it is empty(so only a skeleton file). However, this class will be used to keep data of a specific organization. We added some more content to this file